### PR TITLE
Warp website v2

### PR DESCRIPTION
**Pull request by rexwangcc**
To preview locally, from the root of the repo:

```
yarn --cwd=website install
```

and then:

```
yarn --cwd=website start
```

### CODE REVIEW

**Comment by kbergin** [comment]
I noticed there's a new contribution guide section - I am looking forward to fleshing this out with information on how to contribute to WARP. For now it seems like 2/3 sub docs in that section are related to contributing to documentation/doc site? Can we make further subcategorizations like

Contributions
-> Welcome
-> Contributing to WARP
----> A
----> B
-> Contributing to this doc site
----> A
----> B

Or something?

**Comment by ekiernan** [comment]
> I noticed there's a new contribution guide section - I am looking forward to fleshing this out with information on how to contribute to WARP. For now it seems like 2/3 sub docs in that section are related to contributing to documentation/doc site? Can we make further subcategorizations like
> 
> Contributions
> -> Welcome
> -> Contributing to WARP
> ----> A
> ----> B
> -> Contributing to this doc site
> ----> A
> ----> B
> 
> Or something?

@kbergin Yes, this should be possible by creating new folders within the Contributions folder.

**Comment by rexwangcc** [comment]
> I noticed there's a new contribution guide section - I am looking forward to fleshing this out with information on how to contribute to WARP. For now it seems like 2/3 sub docs in that section are related to contributing to documentation/doc site? Can we make further subcategorizations like
> 
> Contributions
> -> Welcome
> -> Contributing to WARP
> ----> A
> ----> B
> -> Contributing to this doc site
> ----> A
> ----> B
> 
> Or something?

Updated in the latest commit, the preview should reflect it!

File: website/src/pages/privacy.md [hunk 1]
```diff
@@ -13,3 +13,5 @@ Above this page is not a legal notice, please refer to [Google Analystics](https
 - Authorized staff on the WARP development team.
 
 - Google Analytics, which WARP docsite relies on to [analyze service usage](https://www.google.com/analytics/terms/us.html). To learn more about how to opt out of Google’s use of cookies as part of its Analytics services, you may visit Google’s Ads Settings. You will be able to prevent your browsing data from being used by Google Analytics by installing Google’s opt-out browser add-on from the Google Analytics Opt-out Page.
+
+- Hotjar, which WARP docsute uses to collect your feedback. You may visit [Hotjar compliance page](https://help.hotjar.com/hc/en-us/categories/360003405813) to learn more about how they operate.
```
Comment (ekiernan) [inline website/src/pages/privacy.md:17]:
```suggestion
Hotjar, which the WARP doc site uses to collect your feedback. You may
```

File: website/src/pages/privacy.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 On this site, we place cookies on your browser to make our system faster.
 
-The only plugin this site relies on to track the traffic is [@docusaurus/plugin-google-analytics](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-analytics), we only use the traffic data provided by [Google Analystics](https://analytics.google.com/) to make this site better and faster.
+The only plugins this site relies on to track the traffic are [@docusaurus/plugin-google-analytics](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-analytics) and [docusaurus-plugin-hotjar](https://github.com/ArtFlag/docusaurus-plugin-hotjar), we only use the traffic data provided by [Google Analystics](https://analytics.google.com/) and [Hotjar](https://www.hotjar.com/) to collect feedback from you and make this site better and faster.
```
Comment (ekiernan) [inline website/src/pages/privacy.md:5]:
```suggestion
The only plugins this site relies on to track the traffic are [@docusaurus/plugin-google-analytics](https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-google-analytics) and [docusaurus-plugin-hotjar](https://github.com/ArtFlag/docusaurus-plugin-hotjar). We only use the traffic data provided by [Google Analystics](https://analytics.google.com/) and [Hotjar](https://www.hotjar.com/) to collect feedback from you and make this site better and faster.
```

**Review by ekiernan — APPROVED** [review]
@rexwangcc LGTM! Thank you again! Are you still planning to add the custom css changes for tip and warning boxes before we merge? I can also reach out to Terra folks after merging.

**Comment by rexwangcc** [comment]
> @rexwangcc LGTM! Thank you again! Are you still planning to add the custom css changes for tip and warning boxes before we merge? I can also reach out to Terra folks after merging.

Thanks! I think we can merge now and tweak the styling later, as this change set is already large and blocking further documentation changes. What's more, I want to make sure the Github Action updates work. (Will have a followup PR to fix them if it does not)

**Review by kbergin — APPROVED** [review]

**Review by gbggrant — APPROVED** [review]

**Pull request merged**